### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by sdodson**
This also details that master branch is 4.0

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+TEMPLATE
+========
+The pull request title should be one line summary that completes the sentance
```
Comment (vrutkovs) [inline .github/PULL_REQUEST_TEMPLATE.md]: typo `sentence`

Comment (vrutkovs) [reply .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
The pull request title should be one line summary that completes the sentence
```

**Comment by DanyC97** [comment]
@sdodson imo will be also good to add a note saying that the PR title should include `[release-3.x]` or `[release-4.x]`.

Imo it will be easier to filter should you need to based on keyword

**Pull request merged**